virttest/utils_windows/drive.py

```diff
@@ -86,3 +86,23 @@ def shrink_volume(session, vol_id, size):
     shrink_cmd += 'echo shrink desired=%s >> {0} ' % size
     shrink_cmd += '&& diskpart /s {0}'
     session.cmd(shrink_cmd.format(script_path))
+
+
+def get_disk_property_by_serial_number(session, serial_number, property_name):
+    """
+    Get disk drive property value by serial number.
+
+    :param session: Windows VM session.
+    :type session: aexpect.ShellSession
+    :param serial_number: Serial number of disk drive.
+    :type serial_number: str
+    :param property_name: The name of property to be get.
+    :type property_name: str
+    :return: The value of property.
+    :rtype: str
+    """
+    cond = "SerialNumber like '%s'" % serial_number
+    cmd = wmic.make_query('diskdrive', cond, props=[property_name],
+                          get_swch=wmic.FMT_TYPE_LIST)
+    return utils_misc.wait_for(
```

Comment:
why we need to wait for ?

Reply:
Need to wait for times since can not get info immediately under the low resource.